test/BuildTools.Tasks.Tests/ZipArchiveTest.cs

```diff
@@ -92,6 +92,40 @@ public void CreatesZip()
             }
         }
 
+        [Fact]
+        public void FailsForEmptyFileName()
```

Comment:
I discovered this bug it while using the ZipArchive task to produce the korebuild artifact.